.github/workflows/playwright.yml

```
@@ -0,0 +1,57 @@
+    paths:
+      - "LLIO2025/**"
+      - "playwrightTests/**"
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)